arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -0,0 +1,715 @@
+      "get": {
+        "summary": "List all of the Media Services in a resource group.",
+        "operationId": "MediaServices_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "Name of the resource group within the Azure subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],        
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MediaServiceCollection"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+            "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}": {
+      "get": {
+        "summary": "Get a Media Service.",
+        "operationId": "MediaServices_GetMediaService",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MediaService"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "Name of the resource group within the Azure subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "mediaServiceName",
+            "description": "Name of the Media Service.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z0-9]",
+            "minLength": 3,
+            "maxLength": 24
+          }
+        ]
+      },
+      "put": {
+        "summary": "Create a Media Service.",
+        "operationId": "MediaServices_PutMediaService",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MediaService"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/MediaService"

```

Comment:
Does the Service follow a workflow where the client is supposed to poll till the provisioningState received in 201 reaches a terminal state? If so then this is a long running operation. Please add the x-ms-long-running-operation extension over here.